### ISSUE TITLE
Sign language displays incorrect names in chat

### ISSUE THREAD

**Issue by Talkarcabbage** [Aurorastation/Aurora.3#91]
It appears that when someone says something in sign language, the name of the observer is used instead of the name of who said it. 
For instance, given my ghost who was never in the round, it displays the name of my ghost as the source of the message
http://puu.sh/n2pGL/233321b4d6.png

**Comment by K0NFL1QT** [comment]
Further to this, using Sign Language outputs text that isn't received by any listeners, even if they have Sign Language too. The 'speaker' sees it, but no-one else does. Also, having Sign Language makes you occasionally gesture "what did i do", without any player input. Thankfully it's not viewable by anyone else anyway.

**Comment by skull132** [comment]
@K0NFL1QT - As we're testing, listeners _are_ hearing what's said, but it is displayed as if the listener is talking. Were there any specific circumstances under which listeners were unable to hear what was being spoken?

As for the automated saying bug, thaat's interesting. Is the output text always "what did i do"?

**Comment by skull132** [comment]
Update. It's actually that you're seeing the effects of the original bug, and interpreting that as your character seeing stuff.

**Comment by K0NFL1QT** [comment]
In a round a few ago, there was a mute skrell arrested and put in the brig. Me and three officers couldn't see their gestures, nor did it appear that they could see mine. As for the messages, it wasn't only  "what did i do". I got that one a few times, but after I reported it I started getting other messages.

Of course, I could have been getting trolled by another sign language user so it showed up, as you say, as if it was said by me.
